bin/run-browser-test-env

```diff
@@ -12,45 +12,73 @@ if [ -f ".secrets" ]; then
   . .secrets
 fi
 
+ci="no"
```

Comment:
doc the flags at the top of the file, in particular so abbreviations like ci are understandable and that azure/aws are mutually exclusive.

Reply:
done